--- a/data/org.gnome.design.AppIconPreview.metainfo.xml.in.in	2020-05-21 14:55:43.350420104 +0200
+++ b/data/org.gnome.design.AppIconPreview.metainfo.xml.in.in	2020-05-21 15:46:05.060820148 +0200
@@ -18,7 +18,7 @@
   </kudos>
   <screenshots>
     <screenshot type="default">
-      <image height="642" width="1056">https://gitlab.gnome.org/World/design/app-icon-preview/raw/master/data/screenshots/screenshot1.png</image>
+      <image height="610" width="1062">https://gitlab.gnome.org/World/design/app-icon-preview/raw/master/data/screenshots/screenshot1.png</image>
       <caption>Previewing an Application icon</caption>
     </screenshot>
   </screenshots>
